app/javascript/utils/form/Field.js

```diff
@@ -24,6 +24,31 @@ export const Field = ({ formApi, field, label, blockNote, errorMessage, children
   )
 }
 
+export const MultiDateField = ({ formApi, field, label, id, blockNote, errorMessage, children }) => {
```

Comment:
I feel this component should be in its own file, like `MultiDateField.js`

Reply:
done!